persistence/persistence-jdbc/src/main/java/uk/gov/justice/services/jdbc/persistence/PaginationCapableRepository.java

```diff
@@ -4,5 +4,13 @@
 import java.util.stream.Stream;
 
 public interface PaginationCapableRepository<E> {
-    Stream<E> getPage(final long offset, final long pageSize, final Map<String, Object> params);
+    Stream<E> getFeed(final long offset,
+                      final Link link,
+                      final long pageSize,
+                      final Map<String, Object> params);
+
+    boolean recordExists(final long offset,
```

Comment:
i'm not sure we need all 4 params..

Reply:
Remove page size and link which is not required